src/components/fileviewer.js

```diff
@@ -33,29 +33,37 @@ export default class FileViewerContainer extends Component {
       });
     });
 
-    FetchAPI.getFileRevisionCoverage(revision, path,
-      (data) => {
-        console.log(data);
-        this.setState(() => ({ coverage: data }));
-        // TODO remove these log lines
-        console.log(this.state.coverage.data[0][0].source.file.covered);
-        console.log(this.state.coverage.data[0][0].source.file.uncovered);
-        console.log(this.state.coverage.data[0][0].source.file.percentage_covered);
-      }
-    )
+    FetchAPI.getFileRevisionCoverage(revision, path)
+      .then(response => {
+        if (response.status !== 200) {
+          console.log('Error status code' + response.status);
+          return;
+        }
+        response.json().then(data => {
+          this.setState({ coverage: data });
+          // TODO remove these log lines
+          console.log(data);
+          console.log(this.state.coverage.data[0][0].source.file.covered);
+          console.log(this.state.coverage.data[0][0].source.file.uncovered);
+          console.log(this.state.coverage.data[0][0].source.file.percentage_covered);
+        });
+      })
+      .catch(err => {
+        console.log('Fetch error', err);
+      })
   };
 
   render() {
     const { revision, path } = this.props;
     return (
       <div>
-        <FileViewerMeta 
-          revision={revision} 
-          path={path} 
+        <FileViewerMeta
+          revision={revision}
+          path={path}
```

Comment:
Sorry for the extra modified lines, I think my editor automatically remove the trailing space occur in JSX

Reply:
My bad, I should have ran yarn lint on the earlier pull request.  I am using Visual Studio editor and thought that it would automatically do the same for me.

Reply:
Oh sorry, I totally forget. I didn't run yarn lint either.

Reply:
use `?w=1` at the end of a github url to ignore the space changes.

https://github.com/blog/967-github-secrets